.github/workflows/ci.yml

```diff
@@ -3,104 +3,128 @@
 
 name: Yellowbrick CI
 
-on: 
-  # Trigger on push to the primary branches only
+on:
+  # Trigger on push to the primary branches and all versioned tags
   push:
     branches:
       - develop
       - main
-      - master
+    tags:
+      - 'v*'
   # Trigger on pull request always (note the trailing colon)
   pull_request:
 
 jobs:
+  # Test Vanilla Python (using PyPI for dependencies) as a matrix.
```

Comment:
Most of the changes below are reordering to have the PyPI tests first then conda tests, as well as some whitespace and comments.